### ISSUE TITLE
build fails

### ISSUE THREAD

**Issue by trebmuh** [OpenMusicKontrollers/orbit.lv2#2]
Trying to build from head master (5517c49), it fails with:

```
[10/11] cc -Iorbit@sha -I. -I.. -I../props.lv2 -I../netatom.lv2 -I../timely.lv2 -I../varchunk -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=c11 -O3 -D_GNU_SOURCE -g -O2 -fdebug-prefix-map=/build/orbit.lv2-0.0.0+20200924=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -fvisibility=hidden -ffast-math -Wno-unused-variable -MD -MQ 'orbit@sha/orbit_timecapsule.c.o' -MF 'orbit@sha/orbit_timecapsule.c.o.d' -o 'orbit@sha/orbit_timecapsule.c.o' -c ../orbit_timecapsule.c
FAILED: orbit@sha/orbit_timecapsule.c.o 
cc -Iorbit@sha -I. -I.. -I../props.lv2 -I../netatom.lv2 -I../timely.lv2 -I../varchunk -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=c11 -O3 -D_GNU_SOURCE -g -O2 -fdebug-prefix-map=/build/orbit.lv2-0.0.0+20200924=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -fvisibility=hidden -ffast-math -Wno-unused-variable -MD -MQ 'orbit@sha/orbit_timecapsule.c.o' -MF 'orbit@sha/orbit_timecapsule.c.o.d' -o 'orbit@sha/orbit_timecapsule.c.o' -c ../orbit_timecapsule.c
../orbit_timecapsule.c: In function '_path_intercept':
../orbit_timecapsule.c:256:46: error: format not a string literal and no format arguments [-Werror=format-security]
   snprintf(job->file_path, len, handle->state.file_path);
                                 ~~~~~~~~~~~~~^~~~~~~~~~
In file included from ../orbit_timecapsule.c:29:
../varchunk/varchunk.h: In function 'varchunk_new':
../varchunk/varchunk.h:145:2: warning: ignoring return value of 'posix_memalign', declared with attribute warn_unused_result [-Wunused-result]
  posix_memalign((void **)&varchunk, sizeof(varchunk_elmnt_t), total_size);
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by ventosus** [comment]
thanks for reporting.

should be fixed in cea76c6c6e42b936e05647ae7e7961a9f4e8a600

**Comment by trebmuh** [comment]
Confirming the building is fixed.
Thank you very much.
